README.md

```diff
@@ -31,7 +30,7 @@ Before producing any amount of code please have a look at [contribution guidelin
 ## Trademark Disclaimer
 
 Product names, logos, brands and other trademarks referred to within the openHAB website are the
-property of their respective trademark holders. These trademark holders are not affiliated with
-openHAB or our website. They do not sponsor or endorse our materials.
+property of their respective trademark holders.  These trademark holders are not affiliated with
+openHAB or our website.  They do not sponsor or endorse our materials.
 
-Google Play and the Google Play logo are trademarks of Google Inc.
+Google Play&trade; and the Google Play&trade; logo are trademarks of Google Inc.
```

Comment:
Please revert. It is exactly what Google wants: https://play.google.com/intl/en_us/badges/